file_formats/ie_formats/eff_v2.htm

```diff
@@ -195,6 +195,10 @@
               <li>bit 2: Death</li>
               <li>bit 3: Wands</li>
               <li>bit 4: Polymorph</li>
+              <li>bit 10: Ignore primary target – Line AoE projectile doesn't affect end target (EE only)</li>
+              <li>bit 11: Ignore secondary target – Line AoE projectile doesn't affect bystanders (EE only)</li>
+              <li>bit 24: Bypass mirror image (EE only)</li>
+              <li>bit 25: Ignore difficulty (EE only)</li>
```

Comment:
These two are from ToBEx.

Reply:
> These two are from ToBEx.

Which ones exactly? Just `BIT24` and `BIT25`?

Reply:
Yep. Just note it in the parentheses, like we do for item bits.

Reply:
> Yep. Just note it in the parentheses, like we do for item bits.

Done.